package.json

```diff
@@ -2,14 +2,15 @@
   "name": "@manageiq/manageiq-api-mock",
   "version": "0.0.0-development",
   "description": "Mock REST API for ManageIQ",
-  "author": "Red Hat",
+  "author": "Chris Hale / Red Hat",
```

Comment:
I know its tempting, but if ya look across the enterprise we don't add individual authors to products.